RELEASE_PROCESS.md

```diff
@@ -0,0 +1,246 @@
+    - [Major and minor Release](#major-and-minor-release)
+      - [example of releasing `v8.0.0-rc0`](#example-of-releasing-v800-rc0)
+      - [example of releasing `v8.0.0`](#example-of-releasing-v800)
+      - [example of releasing `v8.0.1`](#example-of-releasing-v801)
+    - [backport release](#backport-release)
+      - [example of backport release `v7.0.5`](#example-of-backport-release-v705)
+    - [Test building artifacts](#test-building-artifacts)
+    - [Release notes](#release-notes)
+
+
+This document outlines the release process for Cosmos Hub (Gaia).
+
+Gaia follows [semantic versioning](https://semver.org), but with the following deviations to account for state-machine breaking changes: 
+- Changes that requires upgrade trough governance will result in an increase of the major version X (X.y.z). Note that most likely this entails state-machine breaking changes.
+- Changes that are state-machine breaking but do not require upgrade through governance (e.g., emergency upgrades) will result in an increase of the minor version Y (x.Y.z | x > 0).
+- Changes that are not state-machine breaking will result in an increase of the patch version Z (x.x.Z | x > 0).
+
+**Note**: State-machine breaking changes include changes that impact the amount of gas needed to execute a transaction as it results in a different `apphash` after the code is executed.
+
+Every major release will have a release branch and patch releases will be tagged on this branch. No patch releases have their own branch. (This branch strategy only applies to `v7` and later releases.)
+
+## Major Release Procedure
+
+A _major release_ is an increment of the first number (eg: `v7.1.0` → `v8.0.0`) or the _point number_ (eg: `v7.0.0 → v7.1.0`, also called _point release_). Each major release opens a _stable release series_ and receives updates outlined in the [Major Release Maintenance](#major-release-maintenance) section.
+
+> Note: Generally, PRs should target `main` (expect PRs open via the Github mergify integration). 
+
+* Once the team feels that `main` is feature complete, we create a `release/vY` branch (going forward known a release branch), 
+  where `Y` is the version number, with the patch part substituted to `x` (eg: 8.0.x). 
+  * Update the [GitHub mergify integration](./.mergify.yml) by adding instructions for automatically backporting commits from `main` to the `release/vY` using the `A:backport/vY` label.
+  * **PRs targeting directly a release branch can be merged _only_ when exceptional circumstances arise**.
+* In the release branch prepare a new version section in the `CHANGELOG.md`
+    * All links must point to their respective pull request.
+    * The `CHANGELOG.md` must contain only the changes of that specific released version. 
+      All other changelog entries must be deleted and linked to the `main` branch changelog ([example](TBA)).
+    * Create release notes, in `RELEASE_NOTES.md`, highlighting the new features and changes in the version. 
```

Comment:
TODO: Set up goreleaser. See https://github.com/cosmos/cosmos-sdk/blob/a392bae9b88f491d02cfa1fb5ea89e8d605ef840/.github/workflows/release.yml#L26 and https://github.com/cosmos/cosmos-sdk/blob/a392bae9b88f491d02cfa1fb5ea89e8d605ef840/.goreleaser.yml#L2